### PR TITLE
chore: fix OAS3 support link

### PR DESCRIPTION
**Pull request by nadade**

### CODE REVIEW

**Review by abtris — APPROVED** [review]

**Review by kylef — APPROVED** [review]

**Comment by kylef** [comment]
Thanks! Forgot about this one when changing the repo structure.

File: _posts/2019-02-06-OAS-3.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ or providing valuable feedback in our [issue
 tracker](https://github.com/apiaryio/api-elements.js/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Aopenapi3).
 
 You can track the [current status of supported OAS 3
-functionality](https://github.com/apiaryio/api-elements.js/blob/master/packages/fury-adapter-oas3-parser/STATUS.md),
+functionality](https://github.com/apiaryio/api-elements.js/blob/master/packages/openapi3-parser/STATUS.md),
 we are also publishing a [list of the changes we make to our
 parser](https://github.com/apiaryio/api-elements.js/blob/master/packages/fury-adapter-oas3-parser/CHANGELOG.md)
```
Comment (kylef) [inline _posts/2019-02-06-OAS-3.md:92]:
```suggestion
parser](https://github.com/apiaryio/api-elements.js/blob/master/packages/openapi3-parser/CHANGELOG.md)
```

Comment (kylef) [reply _posts/2019-02-06-OAS-3.md:92]: There's also `fury-adapter-oas3-parser` on line 85, but GitHub won't let me suggest there as its too far out of the diff view.

**Review by abtris — CHANGES_REQUESTED** [review]
Need apply kyle suggested changes.

**Pull request closed without merging**